horovod/common/parameter_manager.cc

```diff
@@ -63,7 +62,7 @@ ParameterManager::ParameterManager() :
   Reset();
 }
 
-void ParameterManager::CreateMpiTypes() {
+void ParameterManager::CreateMpiTypes(MPI_Datatype &mpi_params_type_) {
```

Comment:
Looks like `CreateMpiTypes` and `FreeMpiTypes` are no longer used, so let's get rid of them.

Reply:
Removed.